bin/download-syntax

```diff
@@ -356,6 +357,10 @@ REPOS = (
 # END
 
 
+def json_dump(obj: dict[str, Any], f: TextIO) -> None:
+    json.dump(obj, f, separators=(',', ':'))
```

Comment:
aww my comment didn't make it through -- this can be `json_dump = functools.partial(json.dump, separators=(',', ':'))`

Reply:
Sure